tool_collections/kraken/kraken/kraken.xml

```diff
@@ -9,7 +9,23 @@
     <command>
         <![CDATA[
         kraken --threads \${GALAXY_SLOTS:-1} @INPUT_DATABASE@ 
+        
+        #if isinstance( $input_sequences.datatype, $__app__.datatypes_registry.get_datatype_by_extension('fastq').__class__):
```

Comment:
Yeah - I think what you want is

```
$input_sequences.is_of_type( 'fastq' ) 
```

https://bitbucket.org/galaxy/galaxy-central/pull-request/457/allow-cheetah-tool-templates-to-reason/diff